From 3ad884b3751a13ca2697282ca94cdc7f448b831b Mon Sep 17 00:00:00 2001
From: forumber <kidobeko@gmail.com>
Date: Sun, 15 Nov 2015 15:35:49 +0200
Subject: [PATCH 1/1] fix audio

---
 hal/msm8974/platform.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/hal/msm8974/platform.c b/hal/msm8974/platform.c
index a447803..befc047 100644
--- a/hal/msm8974/platform.c
+++ b/hal/msm8974/platform.c
@@ -944,12 +944,6 @@ void *platform_init(struct audio_device *adev)
             ALOGE("%s: Could not find the symbol acdb_loader_deallocate_ACDB from %s",
                   __func__, LIB_ACDB_LOADER);
 
-        my_data->acdb_send_audio_cal = (acdb_send_audio_cal_t)dlsym(my_data->acdb_handle,
-                                                    "acdb_loader_send_audio_cal");
-        if (!my_data->acdb_send_audio_cal)
-            ALOGE("%s: Could not find the symbol acdb_send_audio_cal from %s",
-                  __func__, LIB_ACDB_LOADER);
-
         my_data->acdb_send_voice_cal = (acdb_send_voice_cal_t)dlsym(my_data->acdb_handle,
                                                     "acdb_loader_send_voice_cal");
         if (!my_data->acdb_send_voice_cal)
-- 
2.1.4

